src/main.py

```diff
@@ -116,6 +116,9 @@ async def knative_request(
         The headers to pass to knative.
     body: `bytes`
         The next visit message body.
+    info: `str`
+        Information such as some data IDs of the next visit message to identify
+        this request and to log with.
```

Comment:
Maybe "some fields" instead of "some data IDs"? We're already used to "data ID" meaning something more specific.